libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -18,31 +18,55 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-/* 
+/*
    FRSKY Telemetry library
 */
 
-#include "AP_Frsky_Telem.h"
-
 #include <AP_AHRS/AP_AHRS.h>
 #include <AP_BattMonitor/AP_BattMonitor.h>
 #include <AP_RangeFinder/AP_RangeFinder.h>
 #include <AP_Common/AP_FWVersion.h>
-#include <GCS_MAVLink/GCS.h>
 #include <AP_Common/Location.h>
 #include <AP_GPS/AP_GPS.h>
 #include <AP_Baro/AP_Baro.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Logger/AP_Logger.h>
+#include <AP_Vehicle/AP_Vehicle.h>
+#include <AC_Fence/AC_Fence.h>
+#include <AP_HAL/utility/sparse-endian.h>
+
+#include <stdio.h>
 #include <math.h>
 
+#include "AP_Frsky_Telem.h"
```

Comment:
Usually this should go above `stdio.h and `math.h`

Reply:
ok, so the general approach should be from local to global
- this class header
- local headers
- AP headers
- standard headers

all in alphabetical order?